FHEM/00_SIGNALduino.pm

```diff
@@ -38,7 +38,7 @@ use List::Util qw(first);
 
 
 use constant {
-  SDUINO_VERSION                  => '3.5.4+20220125',  # Datum wird automatisch bei jedem pull request aktualisiert
+  SDUINO_VERSION                  => '3.5.2+20220129',  # Datum wird automatisch bei jedem pull request aktualisiert
```

Comment:
Hier bitte anpassen :)

Reply:
Ich war mir eigentlich sicher, das ich daran nichts geändert habe, weil du es ja schon hier geändert hattest: https://github.com/RFD-FHEM/RFFHEM/commit/6c08ac49c6980aaa4f09300a3d3907b8a21ebadc

Eben habe ich nochmal einen commit durchgeführt: https://github.com/RFD-FHEM/RFFHEM/pull/1072/commits/5494de7828fd5bb249e2ae35c1674b36f5ed50f7
aber die Version wird offensichtlich anschließend wieder zurück gesetzt: https://github.com/RFD-FHEM/RFFHEM/pull/1072/commits/1b1549c61c6cb4be10d952926debcbec899d819a

Reply:
Ich habe es hart eincodiert..... :(

Reply:
Ich habe in #1073 eine Anpassung vorgenommen, so dass die Version von einer anderen Stelle als Referenz verwendet wird.